README.md

```diff
@@ -1 +1,33 @@
-# ES6-AwesomeBooks
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+In this project we build a aplication called Awesome Books where you can create a list of your favorites books, this aplication was built with plenty JavaScript to add and remove elements from the DOM and store data in the local storage.
+
+## Built With
+
+- Major languages
+- HTML, CSS, Javascript
+- Frameworks
+
```

Comment:
- [ ] Please it will be nice to add a `Getting started` section in your `Readme file` where you outline explicit steps on how to clone and run you app locally, as this will be of help to non technical users.  Please  checkout this [template](https://github.com/microverseinc/readme-template) from Microverse and adapt it to your project.